components/atom/tag/src/Actionable/index.js

```diff
@@ -49,13 +51,14 @@ ActionableTag.propTypes = {
   iconPlacement: PropTypes.oneOf([LEFT_ICON_PLACEMENT, RIGHT_ICON_PLACEMENT]),
   onClick: PropTypes.func,
   target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top']),
+  rel: PropTypes.oneOf(['nofollow', 'noopener', 'noreferrer']),
```

Comment:
W3C spec says that rel attribute value is a space-separated list of link types so it can have 1 or more of link types.

https://www.w3.org/TR/html401/struct/links.html#adef-rel